docs/contributing/index.md

```diff
@@ -1,69 +1,93 @@
 
-- Find a bug and open a new issue.
-- Propose an idea of features that will enhance Autoware.
-- Join an existing discussion and express your opinion.
-- Organize discussions for other contributors.
-- Answer questions and support other contributors.
+To see the schedule for upcoming working group meetings, refer to the [Autoware Foundation events calendar](https://calendar.google.com/calendar/u/0/embed?src=autoware.org_6lol0ho5ft0217h8c60pi1fm30@group.calendar.google.com).
 
-See our [support guidelines](../help/support-guidelines.md) for the detailed steps.
+### Reporting bugs
+
+Before you submit an issue, please search the issue tracker for the appropriate repository. It is possible that someone has already reported the same issue and that workarounds exist. If you can't determine the appropriate repository, ask the maintainers for help by creating a new discussion in the [Q&A category](https://github.com/autowarefoundation/autoware/discussions/categories/q-a).
+
+In order to fix a bug, we need to reproduce and confirm it first and so you should provide a minimal reproduction. Having a minimal reproducible scenario allows us to quickly confirm a bug (or point out a coding or configuration problem) as well as confirm that we are fixing the right problem.
+
+If you want to fix the bug by yourself, then that will be appreciated, but you should discuss possible approaches with the maintainers first before submitting a pull request.
+
+You can file new issues by selecting from our new issue templates and filling out the issue template.
 
 ### Pull requests
 
-You can submit a pull request directly (without opening an issue) if it is small like the following:
+Pull requests can be submitted directly for small changes such as:
+
+- Minor documentation updates.
+- Fixing spelling mistakes.
+- Fixing CI failures.
+- Fixing warnings detected by compilers or analysis tools.
+- Making small changes to a single package.
 
-- Update a small part of documents.
-- Fix spelling mistakes.
-- Fix CI failures.
-- Fix warnings detected by compilers or analysis tools.
-- Make small changes to a single package.
+If your pull request is not covered by the examples above or is a large change, create an issue or a discussion thread to discuss possible approaches with the maintainers before submitting a pull request. Doing so ensures that the proposed change will be in line with Autoware's design philosophy and current development plans. Lastly, ensure that you use the appropriate pull request template for the target repository.
```

Comment:
I think it is better to include where to discuss.

Reply:
@kenji-miyake Should people use the the Q&A category to ask the maintainers about large changes, or changes that are not covered by the list of small changes?

Reply:
I think we should recommend that people create a Q&A discussion for large changes, so that issues are kept for confirmed bugs only.

```suggestion
If your pull request is a large change not covered by the examples above, create a Q&A discussion thread to confirm with the maintainers before submission. Doing so ensures that the proposed change is in line with Autoware's design philosophy and current development plans. Lastly, ensure that you use the appropriate pull request template for the target repository.
```

Reply:
> @kenji-miyake Should people use the the Q&A category to ask the maintainers about large changes, or changes that are not covered by the list of small changes?

@LalithVipulananthan For newbies, I think yes. Or sometimes they can also use `Design`.
For accustomed contributors, we can use Discord or Issues.

So I think it's good to write like "if you're not sure where to discuss, ask in the Q&A category".